packages/mask/src/plugins/Saving/SNSAdaptor/saving/CompoundMarketItem.tsx

```diff
@@ -0,0 +1,88 @@
+import type { CompoundMarket } from '../../contracts/compound/useCompound'
+import { makeStyles } from '@masknet/theme'
+import { Button } from '@mui/material'
+import { useI18N } from '../../../../utils'
+import { formatBalance } from '@masknet/web3-shared-evm'
+
+const useStyles = makeStyles()((theme) => ({
+    wrap: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+        background: theme.palette.mode === 'light' ? '#F6F8F8' : '#17191D',
```

Comment:
Avoid using dead code color.

```suggestion
        background: palette.background.paper,
```